### DISCUSSION TITLE
'emotes' animations

### DISCUSSION THREAD

**Discussion by cloudziq** [redeclipse/base#1147] [General]
Hello all readers! :)
After creating new crouch and claw2 anims, one idea born in my head - the emotes, ability for players to express yourself in a few ways while playing.
Currently I working on one that have WIP name: "hello hand"

<a href="https://imgur.com/igG5MYx"><img src="https://i.imgur.com/igG5MYxl.gif" title="source: imgur.com" /></a>

This is first animation, still not ideal I think, so any suggestions are welcome. It is just a proof of concept and maybe when (if) this will be included in game and we will see how it works, then we decide about next steps.

anim file itself: (its just for main devs, since this is useless without any implementation)
[hello_hand.zip](https://github.com/red-eclipse/world/files/2144164/hello_hand.zip)

**Comment by bonifarz** [comment]
Well done, Dziq
I think this could end up as a really nice feature, with lots of funny gestures, like taunts, cheers, kicks and facepalms. Might get tricky to blend the animations when the actor is moving, though.

**Comment by cloudziq** [comment]
another one: 
<a href="https://imgur.com/lesVJsB"><img src="https://i.imgur.com/lesVJsBl.gif" title="source: imgur.com" /></a>

**Comment by cloudziq** [comment]
I don't know what to do.. maybe idea is good.. but for facepalm anim there are some problems with vanities like glasses/cap ..  I don't like that hand overlaps with these. :(

**Comment by qreeves** [comment]
Just do the animation at a slight distance away from the face, it won't look as bad this way because it doesn't matter if it touches the face when seen from a distance.

**Comment by cloudziq** [comment]
but this will look bad that hand will be 5cm or more to face.. hmmm still thinking what to do.. cap and mask is a problematic the most
..if only anim can be different based on some vanities. for Now better leave this as it is
